backend/src/openbeheer/catalogi/api/viewsets.py

```diff
@@ -13,3 +19,63 @@
 )
 class ZaaktypenViewSet(ZGWViewSet):
     service_type = APITypes.ztc
+
+
+@extend_schema(
+    summary=_("Get Open Zaak choices"),
+    description=_(
+        "Get the available Open Zaak catalogi instances. The value is the slug of the configured service, "
+        "while the label is the name of the service."
+    ),
+    # responses={
+    #     "200": list[OBOption[str]] # TODO blueprint of msgspec
+    # }
+)
+class ServiceChoicesView(MsgspecAPIView): # TODO rebase on Chris PR
```

Comment:
TODO: Decide whether we are going to support environments with multiple Open Zaak instances.